SU-UComm/stanford-report
# Adjust skip link scroll positions

# READY FOR REVIEW

# Summary
- Adjustments for desktop viewports only (mobile/tablet to be addressed in separate ticket) so when user clicks the skip link they are scrolled to the beginning of the page's content and it isn't hidden under the fixed nav.

# Review By (Date)
- ASAP

# Criticality
- 3

# Review Tasks

## Test story pages + Homepage - Desktop only

1. Basic Story: https://news.stanford.edu/developreport/stories/get-to-know-mariachi-cardenal-de-stanford
2. Feature Story: https://news.stanford.edu/developreport/stories/blending-disciplines-for-a-brighter-world
3. Example Homepage: https://news.stanford.edu/developreport/homepage

For each of the above, if you click within the red empty space to the left of "Stanford University" then Shift + Tab on your keyboard the "Skip to content" link should appear as red linked text on a white background. When you click this, it should scroll the page to the top of the content, generally the top of where the heading or hero area are. These elements should not be overlapped by the site's sticky header but sit just below it.

## Test topic listing pages - Desktop only

1. Pull this branch, or `develop`
2. Open `frontend.html`
3. Comment out lines 29-33 – `feature-story-hero` data set
4. Uncomment lines 24-28 – `subtopic-subnav-component` data set
5. npm start
6. Review the [subtopic-subnav-component](http://localhost:3000/r/stanford-components/subtopic-subnav-component/2.0.0?_previewKey=default)

if you click within the red space at the top of the page then Shift + Tab on your keyboard the "Skip to content" link should appear as red linked text on a white background. When you click this, the page should scroll so that the small, gray parent term is just below the red bar (this simulates the sticky nav) at the top of the page, it should not be underneath it.

# Associated Issues and/or People
[UCP-3308](https://stanfordits.atlassian.net/browse/UCP-3308)

[UCP-3308]: https://stanfordits.atlassian.net/browse/UCP-3308?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
